api/src/test/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+server:
+  port: 8100
+
+dwp:
```

Comment:
do we need the config here? It's the same as the main application.yml.

Reply:
Having the copy here means we don't have to update the tests if we change the production values.